python/dgl/subgraph.py

```diff
@@ -276,7 +274,10 @@ def edge_subgraph(graph, edges, preserve_nodes=False, store_ids=True):
     --------
     node_subgraph
     """
-    if graph.is_block and not preserve_nodes:
+    if len(deprecated_kwargs) != 0:
+        raise DGLError("Key word argument preserve_nodes is deprecated. "
+                       "Use relabel_nodes instead.")
```

Comment:
Suggest raise deprecation warning for this release to have a smoother transition.